resourceIndexer/inventory.js

```diff
@@ -4,12 +4,14 @@ import zlib from 'zlib';
 import {writeFileSync} from 'fs';
 import { promisify } from 'util';
 import AWS from 'aws-sdk';
-import {logger} from './logger.js';
 import { DateTime } from 'luxon';
 import escapeStringRegexp from 'escape-string-regexp';
 import { ResourceIndexerError } from './errors.js';
+import debugFactory from 'debug';
+const debug = debugFactory("resource-indexer:inventory");
```

Comment:
Let's keep all of our own codebase's debug logging namespaced under `nextstrain:`? That makes it easy to enable everything of our own with `DEBUG=nextstrain:*`.

```suggestion
const debug = debugFactory("nextstrain:resource-indexer:inventory");
```

Reply:
Yup. Changed in repush (and updated help message also)